docs/integrations/saas-cloud/index.md

```diff
@@ -22,7 +22,7 @@ Learn about the Sumo Logic apps for SaaS and Cloud applications.
   <div className="container">
   <a href="/docs/integrations/saas-cloud/akamai-cloud-monitor">
   <img src={useBaseUrl('img/integrations/saas-cloud/akamai.svg')} alt="icon" width="85"/><h4>Akamai Cloud Monitor</h4></a>
-  <p>A guide to the Sumo Logic App for Akamai Cloud Monitor.</p>
+  <p>A guide to our Akamai Cloud Monitor app.</p>
```

Comment:
As much as I would love this to be the style. Legal and Brand have asked us to make it the Sumo Logic App for X.

Reply:
Oh ok, thanks for letting me know. I'd cut the wording because it was too long and threw off card alignment. I'll try cutting or shrinking the header. @stacykor